app/scripts/app.js

```diff
@@ -0,0 +1,253 @@
+                  files:['bower_components/angular-sanitize/angular-sanitize.js']
+                }),
+                $ocLazyLoad.load(
+                {
+                  name:'ngTouch',
+                  files:['bower_components/angular-touch/angular-touch.js']
+                })
+
+
+            }
+        }
+    })
+      .state('dashboard.home',{
+        url:'/home',
+        controller: 'MainCtrl',
+        templateUrl:'views/dashboard/home.html',
+        authenticate: true,
+        resolve: {
+          loadMyFiles:function($ocLazyLoad) {
+            return $ocLazyLoad.load({
+              name:'sbAdminApp',
+              files:[
+              'scripts/controllers/main.js',
+              'scripts/directives/timeline/timeline.js',
+              'scripts/directives/notifications/notifications.js',
+              'scripts/directives/chat/chat.js',
+              'scripts/directives/dashboard/stats/stats.js'
+
+              ]
+            })
+          }
+        }
+      })
+      .state('dashboard.form',{
+        templateUrl:'views/form.html',
+        url:'/form'
+    })
+      .state('dashboard.blank',{
+        templateUrl:'views/pages/blank.html',
+        url:'/blank'
+    })
+      .state('login',{
+        templateUrl:'views/pages/login.html',
+        url:'/login',
+        controller:'LoginCtrl',
+        authenticate: false,
+          resolve: {
+            loadMyFile:function($ocLazyLoad) {
+              return $ocLazyLoad.load({
+                name:'sbAdminApp',
+                files:['scripts/controllers/LoginController.js']
+              }),
+              $ocLazyLoad.load(
+              {
+                 name:'toggle-switch',
+                 files:["bower_components/angular-toggle-switch/angular-toggle-switch.min.js",
+                        "bower_components/angular-toggle-switch/angular-toggle-switch.css"
+                    ]
+              }),
+              $ocLazyLoad.load(
+              {
+                name:'ngAnimate',
+                files:['bower_components/angular-animate/angular-animate.js']
+              }),
+              $ocLazyLoad.load(
+              {
+                name:'ngCookies',
+                files:['bower_components/angular-cookies/angular-cookies.js']
+              }),
+              $ocLazyLoad.load(
+              {
+                name:'ngResource',
+                files:['bower_components/angular-resource/angular-resource.js']
+              }),
+              $ocLazyLoad.load(
+              {
+                name:'ngSanitize',
+                files:['bower_components/angular-sanitize/angular-sanitize.js']
+              }),
+              $ocLazyLoad.load(
+              {
+                name:'ngTouch',
+                files:['bower_components/angular-touch/angular-touch.js']
+              })
+            }
+          }
+
+    })
+      .state('dashboard.chart',{
+        templateUrl:'views/chart.html',
+        url:'/chart?trackid',
```

Comment:
I prefere the REST style of URLs, `/char/:trackid` see also Trello: https://trello.com/c/4bnK07zC/27-url-patterns